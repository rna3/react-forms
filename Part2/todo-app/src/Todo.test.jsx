import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

test("renders Todo without crashing", () => {
  render(<Todo id={1} task="Test Todo" removeTodo={() => {}} />);
});

test("matches snapshot for Todo", () => {
    const { asFragment } = render(<Todo id={1} task="Test Todo" removeTodo={() => {}} />);
    expect(asFragment()).toMatchSnapshot();
  });