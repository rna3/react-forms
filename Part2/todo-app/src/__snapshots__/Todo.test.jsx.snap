// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for Todo 1`] = `
<DocumentFragment>
  <div>
    <span>
      Test Todo
    </span>
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
